applications/vessel-history/src/features/profile/components/Info.tsx

```diff
@@ -159,20 +160,20 @@ const Info: React.FC<InfoProps> = (props): React.ReactElement => {
                 )}
               </div>
               <InfoField
-                vesselName={vessel.shipname ?? ''}
+                vesselName={vessel.shipname ?? DEFAULT_EMPTY_VALUE}
                 label={VesselFieldLabel.builtYear}
-                value={vessel.builtYear ?? ''}
+                value={vessel.builtYear ?? DEFAULT_EMPTY_VALUE}
               ></InfoField>
               <InfoField
-                vesselName={vessel.shipname ?? ''}
+                vesselName={vessel.shipname ?? DEFAULT_EMPTY_VALUE}
                 label={VesselFieldLabel.owner}
-                value={vessel.owner ?? ''}
+                value={vessel.owner ?? DEFAULT_EMPTY_VALUE}
```

Comment:
as you are doing {value.length > 0 ? value : DEFAULT_EMPTY_VALUE} in InfoField, all this default are not needed here

Reply:
Good catch, I'll revert the change on those lines